.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,107 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[name of the chart e.g. prometheus-kube-stack] Issue Title"
+labels: [bug]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please be cautious with the sensitive information/logs while filing the issue.
+  - type: textarea
+    id: desc
+    attributes:
+      label: Describe the bug a clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: input
+    id: helm-version
+    attributes:
+      label: What's your helm version?
+      description: Enter the output of `$ helm version`
+      placeholder: Copy paste the entire output of the above 
+    validations:
+      required: true
+  - type: input
+    id: kubectl-version
+    attributes:
+      label: What's your kubectl version?
+      description: Enter the output of `$ kubectl version`
+    validations:
+      required: true
+
+  - type: input
+    id: chart-name
+    attributes:
+      label: Which chart?
+      description: Enter the name of the chart where you encountered this bug.
+    validations:
+      required: true
+
+  - type: input
+    id: chart-version
+    attributes:
+      label: What's the chart version?
+      description: Enter the version of the chart that you encountered this bug.
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Enter exactly what happened.
+    validations:
+      required: false
+
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What you expected to happen?
+      description: Enter what you expected to happen.
+    validations:
+      required: false
+
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: How to reproduce it?
+      description: As minimally and precisely as possible.
+    validations:
+      required: false
+
+  - type: textarea
+    id: changed-values
+    attributes:
+      label: Enter the changed values of values.yaml?
+      description: Please enter only values which differ from the defaults. Enter `NONE` if nothing's changed.
+      placeholder: 'key: value'
+    validations:
+      required: false
+
+  - type: input
+    id: helm-command
+    attributes:
+      label: Enter the command that you execute and failing/misfunctioning.
+      description: Enter the command as-is as how you executed.
+      placeholder: helm install my-release prometheus-community/name-of-chart --version version --values values.yaml
+    validations:
+      required: true
+
+
+  - type: textarea
+    id: after-values
+    attributes:
+      label: Enter the Helm values set after installation/upgrade.
+      description: Pate the output of the command `helm get values my-release`
+    validations:
+      required: false
```

Comment:
do we really need it? I mean we have `changed-values` section above it should be sufficient to reproduce the issue. If people are too lazy to have a minimal example to be able to reproduce it then I am asking myself if I should invest the time to figure out how to reproduce it.

Reply:
I agree. I will remove that. 

I was just porting over our markdown prompts to these forms ;)

Reply:
done. removed the `after-values` prompt.